### PR TITLE
feat: update base component

### PR DESCRIPTION
**Pull request by zainab-amir**
https://2u-internal.atlassian.net/browse/VAN-950

Updated Base Component. Changes include:
- Refactored code to rely on paragon classes
- Removed svg yellow line from logistration view
- Fixed yellow line for welcome page
- Updated logo for welcome page

|BEFORE | AFTER |
|------|--------|
Screen Shot 2022-08-18 at 1 10 30 PM|Screen Shot 2022-08-17 at 8 34 57 PM
Screen Shot 2022-08-18 at 12 49 55 PM|Screen Shot 2022-08-18 at 12 49 34 PM
Screen Shot 2022-08-18 at 12 51 18 PM|Screen Shot 2022-08-18 at 12 51 12 PM
Screen Shot 2022-08-18 at 12 51 40 PM|Screen Shot 2022-08-18 at 1 16 13 PM

### CODE REVIEW

File: src/base-component/AuthMediumLayout.jsx [hunk 1]
```diff
@@ -2,68 +2,45 @@ import React from 'react';
...
 
-const AuthMediumLayout = (props) => {
-  const { intl, username } = props;
-
-  return (
-    <div className="container row p-0 mb-3 medium-container">
-      <div className="col-md-10 p-0 screen-header-light">
+const AuthMediumLayout = ({ intl, username }) => (
+  <>
+    <div className="w-100 large-screen-top-stripe large-screen" />
```
Comment (shafqatfarhan) [inline src/base-component/AuthMediumLayout.jsx:12]: Is this large screen's stripe element intentional? If yes, I could not understand its use case here.

File: src/base-component/SmallLayout.jsx [hunk 2]
```diff
@@ -6,38 +6,22 @@ import { Hyperlink, Image } from '@edx/paragon';
...
-              </span>
-            </h1>
-          </div>
-        </div>
-      </div>
-    </>
-  );
-};
+const SmallLayout = ({ intl }) => (
+  <span className="bg-primary-400 vw-100 small-screen">
```
Comment (shafqatfarhan) [inline src/base-component/SmallLayout.jsx:10]: can we use w-100 here to name it consistent with other sizes components?

**Review by shafqatfarhan — APPROVED** [review]
Left a few suggestions, otherwise looks good!

File: src/base-component/AuthMediumLayout.jsx (hunk 1 above)
Comment (zainab-amir) [reply src/base-component/AuthMediumLayout.jsx:12]: It's the original classname. I will update it to `medium-screen-top-stripe`

File: src/base-component/BaseComponent.jsx [hunk 3]
```diff
@@ -17,51 +17,21 @@ import SmallLayout from './SmallLayout';
...
-          {authenticatedUser ? <AuthMediumLayout username={authenticatedUser.username} /> : (
-            <MediumLayout />
-          )}
-        </MediaQuery>
-        <MediaQuery minWidth={breakpoints.large.minWidth} maxWidth={breakpoints.large.maxWidth}>
-          <div className="w-100 large-screen-top-stripe" />
-          {authenticatedUser ? <AuthMediumLayout username={authenticatedUser.username} /> : (
-            <MediumLayout />
-          )}
+      <div className="col-md-12 extra-large-screen-top-stripe extra-large-screen" />
```
[the remark this answers is not in the document]
Comment (zainab-amir) [reply src/base-component/BaseComponent.jsx:25]: That will not be possible. This stripe needs to outside the `layout` block and other stripes need to be inside the layout block.

File: src/base-component/AuthMediumLayout.jsx (hunk 1 above)
Comment (zainab-amir) [reply src/base-component/AuthMediumLayout.jsx]: Simplified the classes.

File: src/base-component/BaseComponent.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (shafqatfarhan) [reply src/base-component/BaseComponent.jsx]: Got it. By the way, on your branch, I tweaked the logic and tried it myself locally and pulled all the stripe elements here before the layout block and it was working. The default state of the `display` property was none and conditional bootstrap breakpoints made required the `display` to flex. Plus point, it grouped the desired logic in one place.

**Pull request merged**